src/book/mod.rs

```diff
@@ -190,6 +204,13 @@ impl MDBook {
         self
     }
 
+    /// You can add a new preprocessor by using this method.
+    /// The only requirement is for your renderer to implement the Preprocessor trait.
```

Comment:
This doc-comment should probably be reworded in the "imperative" tense to describe what it does. So something like "Register a \[`Preprocessor`\] to be used when rendering the book", with the word "Preprocessor" being a link to the relevant trait so people can see the required methods and what types already implement `Preprocessor`.